tests/test_middleware_fastapi.py

```
@@ -0,0 +1,297 @@
+
+
+def _login_and_get_state(tc):
+    """Drive /login through the test client to obtain a valid state cookie,
+    exactly as a real browser would before hitting /callback."""
+    tc.get("/login", follow_redirects=False)
+    return tc.cookies.get("sk_oauth_state")
+
+
+class TestScalekitAuthFastAPI(unittest.TestCase):
+    def test_login_redirects_to_authorization_url(self):
+        app, auth, client = _build_app()
```

Ruff (0.16.1)

[warning] Unpacked variable `auth` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)